frontend/src/pages/spotnet/dashboard/add_deposit.jsx

```diff
@@ -0,0 +1,55 @@
+
+import React,{useState} from 'react';
+import Card from 'components/Card/Card';
+import './add_deposit.css';
+import TokenSelectorDeposit from 'components/TokenSelector/TokenSelector';
+import { ReactComponent as HealthIcon } from 'assets/icons/health.svg';
+import { ReactComponent as EthIcon } from 'assets/icons/ethereum.svg';
+export default function Add_Deposit() {
+   const [amount, setAmount] = useState('0');
+  const handleAmountChange = (e) => {
+    const value = e.target.value;
+    const regex = /^\d*\.?\d*$/;
```

Comment:
please extract this to a `utils/regex.js` folder and name as a constant with an uppercase with more meaningful name, like `NUMBER_REGEX` or smth